templates/base.html

```diff
@@ -226,7 +226,7 @@
 
 <div class="printer-only printer-only-footer">
     <div class="printer-only-user">
-        {{content.stamp.createdBy}}
+        Generated by: {{content.stamp.createdBy}}
     </div>
     <div class="printer-only-time-stamp">
         {{content.stamp.createdAt}}
```

Comment:
@callkalpa We need this too to be the generated time

Reply:
@dinukadesilva made changes to show generated user and generated timestamp.

Timestamp is not when the user clicks on Print in frontend (print is only a frontend thing and there are no API calls), rather it is when the template is rendered from the backend.